package VehiclePackage;

message Vehicle {
	optional double obj = 1;
}
